harvester/node-manager
# Add ksm merge-across-nodes

Problem:
User machines need to support KSM merge-across-nodes

Solution:
Add KSM merge-across-nodes

Related Issue:
https://github.com/harvester/harvester/issues/2827

Test plan:
1. Toggle ksmtuned merge_across_nodes, `kubectl edit ksmtuned node1`, update **spec.ksmtunedParameters.MergeAcrossNodes** to 0 or 1.
2. Wait for a few minutes, `cat /sys/kernel/mm/ksm/merge_across_nodes` should display you want.
